minimal/compose/certbot/production/certbot-renew

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-/opt/certbot/venv/bin/certbot certonly --webroot -n --agree-tos -w /code/staticfiles -m ${EMAIL} -d ${DOMAIN} >> /var/log/certbot-renew.log 2>&1
+/opt/certbot/venv/bin/certbot renew --allow-subset-of-names -q >> /var/log/certbot-renew.log 2>&1
```

Comment:
We can test this by attaching to the running container and issuing `/opt/certbot/venv/bin/certbot renew --allow-subset-of-names --dry-run`. This will simulate and almost expiring certificate. If you're referring to the cron job, we can change the renew script and the time at which it runs, then wait a while to see the output in the logs. I'll test this once I've managed to test the other issues we spoke about.